File: pset6/mario.py
#!/usr/bin/env python3

import cs50

def main():
    print('Please enter the height of your pyramid! ', end = '')
    height = cs50.get_int()
    
    while height > 23 or height < 0:
        print('Please enter a height between 0 and 23: ', end = '')
        height = cs50.get_int()
        
    for i in range(1, height + 1):
        print(' ' * (height - i) + "#" * i + ' ' + "#" * i)
        
if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>
 
int main(void)
{
    
    int height;
    
    do {
        printf("Please enter the height of your pyramid! ");
        height = get_int();
    } 
    
    while (height > 23 || height < 0); 
    
    for (int i = 0; i < height; i++)
    {
    	for (int j = 0; j < height - i - 1;  j++)
    	{
    		printf(" ");
    	}
    
    	for (int j = 0; j <= i;  j++)
    	{
    		printf("#");
    	}
    
    	printf("  ");
    
    	for (int j = 0; j <= i;  j++)
    	{
    		printf("#");
    	}
    
    	printf("\n");
    }
}